subsys/bluetooth/host/conn.c

```diff
@@ -2835,6 +2835,24 @@ int bt_conn_auth_cb_register(const struct bt_conn_auth_cb *cb)
 	return 0;
 }
 
+int bt_conn_auth_cb_register_conn(struct bt_conn *conn, const struct bt_conn_auth_cb *cb)
+{
+	/* The cancel callback must always be provided if the app provides
+	 * interactive callbacks.
+	 */
+	if (!cb->cancel &&
+	    (cb->passkey_display || cb->passkey_entry || cb->passkey_confirm ||
+	     cb->pairing_confirm)) {
+		return -EINVAL;
+	}
+
+	if (IS_ENABLED(CONFIG_BT_SMP) && conn->type == BT_CONN_TYPE_LE) {
```

Comment:
```suggestion
	if (IS_ENABLED(CONFIG_BT_SMP) && (conn->type == BT_CONN_TYPE_LE)) {
```

Also in other places

Reply:
In general, it seems that the convention is to avoid using brackets in similar cases. Some time ago, I was even requested to remove brackets in similar use-case. See: https://github.com/zephyrproject-rtos/zephyr/pull/40599#discussion_r755442826 for details.